webrender/src/frame.rs

```diff
@@ -383,17 +380,20 @@ impl Frame {
 
         // If we have a transformation, we establish a new reference frame. This means
         // that fixed position stacking contexts are positioned relative to us.
-        if stacking_context_transform != LayoutTransform::identity() ||
-           stacking_context.perspective != LayoutTransform::identity() {
+        if stacking_context.transform.is_some() || stacking_context.perspective.is_some() {
+            let transform =
+                context.scene.properties.resolve_layout_transform(&stacking_context.transform);
```

Comment:
nit: I think usually `Option<&T>` is preferred, given it saves an indirection and gets optimized as a null pointer for the `None` case (you'd need to do `stacking_context.transform.as_ref()` here). That being said, I don't think it matters a lot here.